.terraform-version

```diff
@@ -1 +1 @@
-1.6.3
+1.5.7
```

Comment:
There was a Gruntworks module that in inadvertently upgraded when making a GWRC version of it. I didn't initially realize the "terraform-aws-service-catalog" wasn't pointing to the latest tag. and then found out the latest version had actually pinned terraform to 1.5.7 as max